documentation/ADRs/020_Log_Files_and_Real-Time_Alerts.md

```diff
@@ -0,0 +1,75 @@
+# Log Files and Real-Time Alerts
+
+| ADR Info            | Details                   |
+|---------------------|---------------------------|
+| Subject             | Log files and real-time alerts |
+| ADR Number          | 020                       |
+| Status              | Proposed                  |
+| Author              | Marina                    |
+| Date                | 28.10.2024                |
+
+
+## Context
+In order to have a more effective overview of the orchestration in the pipeline, there is a need for better handling of log files and implementing real-time alerts which would allow for timely and relevant error handling for critical issues. 
+
+
+## Decision
+To implement a real-time alert and notification system which will distribute alerts through specific channels (eg. Slack, email, W&B, Prefect), targeted at particular audiences (eg. Infrastructure, MD&D, individual) depending on the type of alert being handled. 
```

Comment:
This is super, but maybe a bit more detail like:

We will implement a real-time alert and notification system that distributes alerts through designated channels—Slack, Email, Prefect, and Weights & Biases (W&B). Alerts will be targeted to specific audiences such as Infrastructure teams, MD&D (Model Development & Deployment) teams, Outreach teams, and individuals responsible for monthly runs. This targeted approach ensures that each alert reaches the appropriate stakeholders promptly, facilitating efficient issue resolution.